src/main/java/graphics/renderer/DebugRenderer.java

```diff
@@ -59,21 +64,52 @@ public void add(GameObject gameObject) {
 				for (DebugPrimitive primitive : primitives) {
 					for (DebugLine line : primitive.getLines())
 						addLine(line);
-			}
+				}
+		}
+	}
+
+	/**
+	 * Remove a gameObject from this renderer
+	 *
+	 * @param gameObject the gameObject
+	 */
+	@Override
+	public void remove(GameObject gameObject) {
+		for (Component c : gameObject.getComponents()) {
+			DebugPrimitive[] primitives = c.debug();
+			if (primitives != null)
```

Comment:
not a big deal but this should probably be wrapped in { } for readability

Reply:
Done.